Firestore/Example/FuzzTests/FuzzingTargets/FSTFuzzTestFieldPath.mm

```diff
@@ -62,9 +67,18 @@ int FuzzTestFieldPath(const uint8_t *data, size_t size) {
     // created as mutable objects. Returns nil if there is a parsing error.
     NSArray *str_arr3 =
         [NSJSONSerialization JSONObjectWithData:d options:NSJSONReadingMutableContainers error:nil];
+    for (int i = 0; i < str_arr3.count; ++i) {
+      if (![str_arr3[i] isKindOfClass:[NSString class]]) {
```

Comment:
Added an explicit `nil` check (though I think it should be okay, because calling methods on `nil` returns some sort of a zero/empty value).

For now, added converting numbers to strings, left recursive conversion as a TODO.